manifests/install.yaml

```diff
@@ -705,15 +705,6 @@ spec:
                                                       type: integer
```

Comment:
Why is the manifest changing so much? Did we just forget to run the make manifests with the library change?

Reply:
Yea looks like I neglected to do it in this PR: https://github.com/argoproj/argo-rollouts/commit/fb287a463487cf783c18a6a6e20181470e96a14e. It should have came in there.